packages/nuka/src/default-carousel-props.tsx

```diff
@@ -3,12 +3,13 @@ import {
   Alignment,
   InternalCarouselProps,
   ControlProps,
-  D3EasingFunctions,
   ScrollMode,
 } from './types';
 import { NextButton, PagingDots, PreviousButton } from './default-controls';
 import { defaultRenderAnnounceSlideMessage } from './announce-slide';
 
+const easeOut = (t: number) => --t * t * t + 1;
```

Comment:
Is the double negative intentional here? Also, is it fair game to do `t ** 3 + 1`?

Reply:
The double negative is intentional. I found this in a gist, and I believe it's some code-golfing level syntax to do `(t - 1) * (t - 1) * (t - 1) + 1`. But I like your idea of using the exponentiation operator, so I'll switch it to that.